frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -132,7 +173,9 @@ export const VMStatus: React.FC<VMStatusProps> = ({ vm, vmi, vmStatusBundle }) =
 
   let icon: React.ComponentClass | React.FC = UnknownIcon;
 
-  if (isPaused) {
+  if (arePendingChanges) {
+    icon = YellowExclamationTriangleIcon;
+  } else if (isPaused) {
     icon = PausedIcon;
   } else if (status === VMStatusEnum.RUNNING) {
     icon = SyncAltIcon;
```

Comment:
it should go under this if, this component should not show pending changes for other states and prefer errors etc